fix(fido2): tests are failing due to NoClassDefFoundError: javax/xml/bind/annotation/XmlElement

**Describe the bug**

https://jenkins.jans.io/jenkins/job/jans-fido2/3555/testReport/junit/io.jans.fido2.service.sg/FullFlowAndroidTest/testFinishAttestationTwoStepAndroid_String__String__String_/

```
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.jboss.weld.injection.producer.DefaultLifecycleCallbackInvoker.invokeMethods(DefaultLifecycleCallbackInvoker.java:83)
	... 64 more
Caused by: java.lang.NoClassDefFoundError: javax/xml/bind/annotation/XmlElement
	at com.fasterxml.jackson.module.jaxb.JaxbAnnotationIntrospector.<init>(JaxbAnnotationIntrospector.java:137)
	at com.fasterxml.jackson.module.jaxb.JaxbAnnotationIntrospector.<init>(JaxbAnnotationIntrospector.java:124)
	at io.jans.fido2.service.DataMapperService.createJsonMapperWithJaxb(DataMapperService.java:102)
	at io.jans.fido2.service.DataMapperService.jsonMapperWithWrapRoot(DataMapperService.java:115)
	at io.jans.fido2.service.DataMapperService.init(DataMapperService.java:50)
	... 69 more
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.annotation.XmlElement
	... 74 more
```

--- a/jans-fido2/server/pom.xml
+++ b/jans-fido2/server/pom.xml
@@ -140,6 +140,10 @@
             <groupId>org.jboss.resteasy</groupId>
             <artifactId>resteasy-servlet-initializer</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-jaxb-annotations</artifactId>
+        </dependency>
 
         <!-- Weld -->
         <dependency>

